src/styles/blocks/body.scss

```diff
@@ -0,0 +1,18 @@
+.body {
+  margin: 0;
+
+  &__card {
+    box-sizing: border-box;
+    width: 200px;
+    height: 406px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    position: relative;
+
+    background-image: url(../images/imac.jpeg);
```

Comment:
Verify that the image `../images/imac.jpeg` exists at the specified path to ensure the background image is displayed correctly.